content/docs/user-guide/data-management/track-sync-data.md

```diff
@@ -0,0 +1,160 @@
+# Track and Sync Versioned Data
+
+The fundamental workflow of most <abbr>DVC projects</abbr> includes the
+following **basic operations**. These can be performed directly (as we cover
+here) but are sometimes included automatically in advanced workflows, like
+[pipelining] and [experiment management].
+
+[pipelining]: /doc/user-guide/pipelines
+[experiment management]: /doc/user-guide/experiment-management
+
+## Tracking data
+
+DVC is [similar to Git] in this area. To start tracking large files or
+directories, "add" them to DVC with the `dvc add` command. This
+<abbr>caches</abbr> the data and [links it] back to the <abbr>workspace</abbr>
+(hiding it from Git). An accompanying `.dvc` file is created.
+
+To capture changes to tracked data, `dvc add` it again (`dvc commit` will also
+do the trick). This caches the latest data and updates `.dvc` files accordingly.
+
+<admon type="info">
+
+`.dvc` files can be tracked (and [versioned](#versioning-data)) with Git.
+
+</admon>
+
+If you need to move or rename tracked data, use `dvc move`. To stop tracking
+data, use `dvc remove`. To also remove it from the cache, use `dvc gc`. See
+[more details].
+
+Putting it all together, we can get an overview of the data in a project with
+`dvc data status`. This will list changes to DVC-tracked data as well as files
```

Comment:
Not sure why `dvc data status` is not auto-linking (see [in deployment](https://dvc-org-guide-data-mgmt-epdkkq.herokuapp.com/doc/user-guide/data-management/track-sync-data#tracking-data)). I see it in https://github.com/iterative/gatsby-theme-iterative/blob/main/packages/gatsby-theme-iterative/config/prismjs/dvc-commands.js . Cc @iterative/websites